examples/neural_networks/plot_random_neural_network.py

```diff
@@ -0,0 +1,153 @@
+"""
+=====================================================================
+The impact of varying alpha and weight_scale on the decision function
```

Comment:
This should have a name that is more easily searchable in the sample gallery, like "Effect of parameters in RandomBasisFunction" or something.